transaction-service/src/main/kotlin/ua/com/foxminded/restClient/kafka/KafkaProducerService.kt

```diff
@@ -0,0 +1,18 @@
+package ua.com.foxminded.restClient.kafka
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.kafka.core.reactive.ReactiveKafkaProducerTemplate
+import org.springframework.stereotype.Service
+
+@Service
+class KafkaProducerService constructor(
```

Comment:
same comment here, what if your application should publish to several topics?

Reply:
Ok, i did some changes